test_scripts/API/AutoCompleteList/008_ATF_SetGlobalProperties_P_AutoCompleteList_ArrayOfUpperBound.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- -- [SetGlobalProperties] Conditions for SDL  must transfer request in case of valid "autoCompleteList" param 
```

Comment:
@Aleksandriys, please correct summary of requirement to be exactly the same as in Jira:
"[SetGlobalProperties] SDL must transfer request to HMI in case of valid "autoCompleteList" param"
and not:
"[SetGlobalProperties] Conditions for SDL must transfer request in case of valid "autoCompleteList" param"